book/src/developers/contributing/rotation/index.md

```diff
@@ -44,12 +44,16 @@ When you get to the end of your checklist, here are ideas for what to work on ne
 - Respond to [Github Participating Notifications](https://github.com/notifications?query=reason%3Aparticipating)
 - Review PRs that have been stuck for >24 hours
 - Find a [Flamingo Issue](https://github.com/ethereum/trin/issues?q=is%3Aopen+is%3Aissue+label%3Aflamingo) that seems promising, and assign it to yourself (and the project dashbord)
+- Run `cargo outdated`
+  - Pick one or several libraries to upgrade
+  - Post a PR with the new `Cargo.lock`
+  - If `cargo outdated` shows all libraries as up-to-date, try `cargo outdated --aggressive`
 - grep code for `TODO`s. For each one you find:
   - write up issue to explain what needs to be done, and a plan
   - link to the TODO in the code, and any relevant context
   - label issue as Flamingo. If it is not urgent and a good fit, add "Good First Issue"
   - Post a PR to remove the `TODO` from the code, in favor of the issue.
```

Comment:
nit (not a blocker for the PR just wanted to mention my stance) The issue I have with issue's is that they become stale, or code might change, but todo's are always in the right spot. Now that we have a bot which closes issues, long term todo's but be lost if we do this.

Reply:
If we choose to discuss this, it should block or delay merging this PR. It isn't that important, I just found todo's are often higher quality and more useful, when I did my massive look through of things to work on 5 months ago.

Reply:
Well, if those same TODOs had been written up as issues instead, then I think you would have had a different experience, where the issues were the best source of things to work on. :)

Still, I get that this adds some manual work that it might be nice to skip. I added https://github.com/ethereum/trin/issues/932 for a possible improvement.

The stale bot won't touch flamingo issues, so that's not a concern for now. I'm open to the bot solution, but don't think it should block the PR.